src/stumpy_png/png.cr

```diff
@@ -118,11 +111,11 @@ module StumpyPNG
         data_pos += scanline_width + 1
 
         case color_type
-        when 0; Scanline.decode_grayscale(decoded, canvas, y, bit_depth)
-        when 4; Scanline.decode_grayscale_alpha(decoded, canvas, y, bit_depth)
-        when 2; Scanline.decode_rgb(decoded, canvas, y, bit_depth)
-        when 6; Scanline.decode_rgb_alpha(decoded, canvas, y, bit_depth)
-        when 3; Scanline.decode_palette(decoded, canvas, y, palette, bit_depth)
+        when 0 then Scanline.decode_grayscale(decoded, canvas, y, bit_depth)
```

Comment:
This is nice, is it new syntax from a recent crystal version?

Reply:
Nope, it's been there from the very early on (beginning?).